packages/jingoo/jingoo.1.3.0/opam

```diff
@@ -4,14 +4,14 @@ authors: "Masaki WATANABE <lambda.watanabe@gmail.com>"
 homepage: "https://github.com/tategakibunko/jingoo"
 bug-reports: "https://github.com/tategakibunko/jingoo/issues"
 dev-repo: "git+https://github.com/tategakibunko/jingoo.git"
-license: "BSD-3-Clause"
+license: "BSD-3"
```

Comment:
```suggestion
license: "BSD-3-Clause"
```
This has been changed by https://github.com/ocaml/opam-repository/pull/14773 to normalize the license names.